src/index.js

```diff
@@ -1,13 +1,56 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import reportWebVitals from "./reportWebVitals";
+import styled from "styled-components";
+import { Header, Main, Prices } from "./components";
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
+const ingredients = [
+  { name: "Bacon", price: 0.75 },
+  { name: "Cheese", price: 1.7 },
+  { name: "Salad", price: 0.95 },
+  { name: "Pickle", price: 1.25 },
+  { name: "Meat", price: 0.75 },
+];
+let burgerPrice = 0;
+function BurgerApp() {
+  return (
+    <Body>
+      <BodyWraper>
+        <Header />
+        <MainWrapper>
+          <Prices property={ingredients} />
+          <Main prop={burgerPrice} />
```

Comment:
```suggestion
          <Main price={burgerPrice} />
```

Reply:
The name "prop" is not good. It should tell and be more specific about the data to be passed